### ISSUE TITLE
[Feature Request] Alternative to Discord

### ISSUE THREAD

**Issue by Soundtoxin** [modularml/mojo#184]
## Request

I would like to see an alternative to Discord for official communications regarding mojo. Something that is free as in freedom like IRC, XMPP, or Matrix.

### Motivation

Discord is not a good platform for discussion of a programming language, and will scare off potential users who refuse to use it on principle.

### Description and Requirements

As a minimum you could bridge the existing Discord chat to some of the other free protocols listed above, with a channel on Libera for the IRC side, but I think it's worth giving some thought to moving entirely off of Discord as well. Even with a bridge, users from the other side are having things they say show up on Discord where the data will be in the hands of the Discord employees/moderators.

**Comment by LOCNNIL** [comment]
@Soundtoxin Actually, nowdays Discord is pretty much popular amoung Open Source development. This [Discord Site](https://discord.com/open-source) talks about some big communities that discuss about their projects on Discord.

There are others projects like:
- Rust
- Rabbit MQ
- NodeJS
- Angular
- React
And many others.

I think that maybe a complementar alternative to Mojo would be a subreddit or a Forum dedicated website.

**Comment by Soundtoxin** [comment]
I don't believe following in the footsteps of others who made the same mistake justifies it. Discord is centralized and proprietary, a big regression from chat protocols we already had in the past. The number of similar communities on IRC (many programming languages, GNU/Linux distros, and other free software projects) and Matrix is significant as well.
With Mojo being so early in development, I think there's still time to correct this mistake. If too much time passes it will be difficult to move the community. I personally can't imagine using Mojo if the main place to ask for help and talk to the community is on Discord, which I will not be using.

**Comment by lattner** [comment]
WE discussed this and opened github forums for more structured discussions.  Right now, we're overwhelmed with all the things so I don't think we're looking to add /more/ options, but I defer to @goldiegadde 

FYI, many Moduler's are out on holiday for the rest of the week.

**Comment by Soundtoxin** [comment]
A forum is a good idea, neither Discord nor the chats I've mentioned are a replacement for a forum, though, so ideally there would be both still. GNU Guix has a channel on the Libera IRC network as well as a mailing list, and it strikes a nice balance. A forum is a bit like the mailing list half of that. Pine64 also has both a forum and IRC (bridged to other chat protocols like Matrix, Discord, Telegram).

No rush, and thank you for your time so far. I just learned of Mojo recently and wanted to bring this idea up ASAP when I saw in the readme and FAQ that Discord was being used. I do understand that it's unlikely the project will change, but better to put in my two cents before things are too cemented for there to even be a chance.
